src/main/kotlin/com/terraformation/backend/api/LocaleConfig.kt

```diff
@@ -0,0 +1,24 @@
+package com.terraformation.backend.api
+
+import com.terraformation.backend.i18n.Locales
+import java.util.*
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.web.servlet.i18n.AcceptHeaderLocaleResolver
+
+@Configuration
+class LocaleConfig {
+  /**
+   * Make Spring's request handlers look at the HTTP Accept-Language header to set the locale for
+   * the current thread.
+   */
+  @Bean
+  fun localeResolver(): AcceptHeaderLocaleResolver {
+    val resolver = AcceptHeaderLocaleResolver()
+
+    resolver.defaultLocale = Locale.ENGLISH
+    resolver.supportedLocales = Locales.supported
```

Comment:
just curious.. are unsupported locales ignored or an error condition?
do we need to set accept-language in the list of allowed headers somewhere or in a yaml?

Reply:
Unsupported locales will be ignored and the default will be used instead. That's a standard header so it's passed through by default by Nginx.